app/services/distribution_reminder.rb

```diff
@@ -0,0 +1,12 @@
+# This job notifies a Partner that they have a distribution scheduled to be sent in 24 hours
+class DistributionReminder
+  def self.perform(dist_id)
+    distribution = Distribution.find_by(id: dist_id)
+
+    # NOTE: This is being also checked in the DistributionMailer itself
+    return if distribution.nil? || distribution.past? || !distribution.partner.send_reminders
+
+    DistributionMailer.delay_until(distribution.issued_at - 1.day).reminder_email(distribution)
```

Comment:
If we're not including the sidekiq stuff is this still going to work? I thought that `delay_until` was a sidekiq feature?

Reply:
Hey @seanmarcia -- I believe so because `DistributionMailer` inherits from `ApplicationMailer` still (i think sidekiq is inserting itself there). So we are able to still call `delay_until` on `DistributionMailer`.

Just to make sure, I was able to queue a reminder_email with this.

Reply:
Awesome!!